tests/type_errors.pl

```diff
@@ -46,13 +46,13 @@ sub correct {
 
 section("SUBTYPING");
 incorrect('(1:unit)');
-correct('ignore(((blank():source(1,1,1)):source(*,*,*)))');
-incorrect('((blank():source(*,*,*)):source(1,1,1))');
+# correct('ignore(((blank():source(1,1,1)):source(*,*,*)))');
+# incorrect('((blank():source(*,*,*)):source(1,1,1))');
```

Comment:
```suggestion
```

Reply:
Nope, the type `*` does not exist anymore: I have remove `Any`

Reply:
So why not remove the test? 🙂